### PR TITLE
validate BOB

### PR DESCRIPTION
**Pull request by TombLabs**
# Validate [BOB](https://solscan.io/token/HKUAxJzGTeeB6bJsgyiwPVncwoRyv8NBghUYEVzXEj1o)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://twitter.com/BOBINSOLANA/status/1736944758113845363
- Coingecko/ CMC URL (If available): Still working on Coingecko and CoinMarketCap for BOB

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [ ] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.

### CODE REVIEW

**Comment by demarco017** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

**Comment by demarco017** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

 Mint address: HKUAxJzGTeeB6bJsgyiwPVncwoRyv8NBghUYEVzXEj1o
 Symbol: BOB
 Token Decimals: 6
 On-chain metadata matches
 No duplication on Jupiter's strict list

**Comment by Buk-zam** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

Mint address: HKUAxJzGTeeB6bJsgyiwPVncwoRyv8NBghUYEVzXEj1o
Symbol: BOB
Token Decimals: 6
On-chain metadata matches
No duplication on Jupiter's strict list

**Comment by DanielInvitado** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

Mint address: HKUAxJzGTeeB6bJsgyiwPVncwoRyv8NBghUYEVzXEj1o
Symbol: BOB
Token Decimals: 6
On-chain metadata matches
No duplication on Jupiter's strict list

**Comment by alexandr19660201** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

Mint address: HKUAxJzGTeeB6bJsgyiwPVncwoRyv8NBghUYEVzXEj1o
Symbol: BOB
Token Decimals: 6
On-chain metadata matches
No duplication on Jupiter's strict li

**Comment by wildandy** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

Mint address: HKUAxJzGTeeB6bJsgyiwPVncwoRyv8NBghUYEVzXEj1o
Symbol: BOB
Token Decimals: 6
On-chain metadata matches
No duplication on Jupiter's strict list

**Comment by Skefterz-afk** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

Mint address: HKUAxJzGTeeB6bJsgyiwPVncwoRyv8NBghUYEVzXEj1o
Symbol: BOB
Token Decimals: 6
On-chain metadata matches
No duplication on Jupiter's strict list

**Comment by elash89** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

Mint address: HKUAxJzGTeeB6bJsgyiwPVncwoRyv8NBghUYEVzXEj1o
Symbol: BOB
Token Decimals: 6
On-chain metadata matches
No duplication on Jupiter's strict list

**Comment by JElite007** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

Mint address: HKUAxJzGTeeB6bJsgyiwPVncwoRyv8NBghUYEVzXEj1o
Symbol: BOB
Token Decimals: 6
On-chain metadata matches
No duplication on Jupiter's strict list

**Comment by Godo5** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

Mint address: HKUAxJzGTeeB6bJsgyiwPVncwoRyv8NBghUYEVzXEj1o
Symbol: BOB
Token Decimals: 6
On-chain metadata matches
No duplication on Jupiter's strict list

**Comment by MaBo974** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

Mint address: HKUAxJzGTeeB6bJsgyiwPVncwoRyv8NBghUYEVzXEj1o
Symbol: BOB
Token Decimals: 6
On-chain metadata matches
No duplication on Jupiter's strict list.

**Comment by PrometheusPyro** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

Mint address: HKUAxJzGTeeB6bJsgyiwPVncwoRyv8NBghUYEVzXEj1o
Symbol: BOB
Token Decimals: 6
On-chain metadata matches
No duplication on Jupiter's strict list

**Comment by CryptoInti** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

Mint address: HKUAxJzGTeeB6bJsgyiwPVncwoRyv8NBghUYEVzXEj1o
Symbol: BOB
Token Decimals: 6
On-chain metadata matches
No duplication on Jupiter's strict list

**Comment by kram710sol** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

Mint address: HKUAxJzGTeeB6bJsgyiwPVncwoRyv8NBghUYEVzXEj1o
Symbol: BOB
Token Decimals: 6
On-chain metadata matches
No duplication on Jupiter's strict list

**Comment by Aemilius420** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

Mint address: HKUAxJzGTeeB6bJsgyiwPVncwoRyv8NBghUYEVzXEj1o
Symbol: BOB
Token Decimals: 6
On-chain metadata matches
No duplication on Jupiter's strict list

**Comment by ImaDefender** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

Mint address: HKUAxJzGTeeB6bJsgyiwPVncwoRyv8NBghUYEVzXEj1o
Symbol: BOB
Token Decimals: 6
On-chain metadata matches
No duplication on Jupiter's strict list

**Comment by Fnkmasterflx** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

Mint address: HKUAxJzGTeeB6bJsgyiwPVncwoRyv8NBghUYEVzXEj1o
Symbol: BOB
Token Decimals: 6
On-chain metadata matches
No duplication on Jupiter's strict list

**Review by Fnkmasterflx — COMMENTED** [review]
I can hereby attest that the provided information are accurate, and the token is valid.

Mint address: HKUAxJzGTeeB6bJsgyiwPVncwoRyv8NBghUYEVzXEj1o
Symbol: BOB
Token Decimals: 6
On-chain metadata matches
No duplication on Jupiter's strict list

**Comment by hpcol2** [comment]
[Fnkmasterflx](https://github.com/Fnkmasterflx) left a comment
I can hereby attest that the provided information are accurate, and the token is valid.

Mint address: HKUAxJzGTeeB6bJsgyiwPVncwoRyv8NBghUYEVzXEj1o
Symbol: BOB
Token Decimals: 6
On-chain metadata matches
No duplication on Jupiter's strict list

**Comment by hpcol2** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

Mint address: HKUAxJzGTeeB6bJsgyiwPVncwoRyv8NBghUYEVzXEj1o
Symbol: BOB
Token Decimals: 6
On-chain metadata matches
No duplication on Jupiter's strict list

**Comment by Senzify** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

Mint address: HKUAxJzGTeeB6bJsgyiwPVncwoRyv8NBghUYEVzXEj1o
Symbol: BOB
Token Decimals: 6
On-chain metadata matches
No duplication on Jupiter's strict list

**Comment by Josemiracool** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

Mint address: HKUAxJzGTeeB6bJsgyiwPVncwoRyv8NBghUYEVzXEj1o
Symbol: BOB
Token Decimals: 6
On-chain metadata matches
No duplication on Jupiter's strict list

**Comment by 0xApolloX** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

Mint address: **HKUAxJzGTeeB6bJsgyiwPVncwoRyv8NBghUYEVzXEj1o**
Symbol: **BOB**
Token Decimals: **6**
On-chain metadata matches
No duplication on Jupiter's strict list

**Comment by Jerryxo4** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

Mint address: HKUAxJzGTeeB6bJsgyiwPVncwoRyv8NBghUYEVzXEj1o
Symbol: BOB
Token Decimals: 6
On-chain metadata matches
No duplication on Jupiter's strict list

**Comment by Exstep** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

Mint address: HKUAxJzGTeeB6bJsgyiwPVncwoRyv8NBghUYEVzXEj1o
Symbol: BOB
Token Decimals: 6
On-chain metadata matches
No duplication on Jupiter's strict list

**Comment by Sumanths1312** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

Mint address: HKUAxJzGTeeB6bJsgyiwPVncwoRyv8NBghUYEVzXEj1o
Symbol: BOB
Token Decimals: 6
On-chain metadata matches
No duplication on Jupiter's strict list

**Comment by kkk075** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

Mint address: facUEejQjHtZJGt2mxYUkiWEPKpDSu9NiBQj2QDqsdZ
Symbol: LTURT
Token Decimals: 0
On-chain metadata matches
No duplication on Jupiter's strict list

File: validated-tokens.csv [hunk 1]
```diff
@@ -1039,4 +1039,5 @@ Rocky,ROCKY,4icEZCrEYNop2ZaMMCkRHaNzkt6xG9BpijMCQV7mpw6Z,9,https://i.ibb.co/X8r9
 Dogelon (Wormhole),ELON,7ZCm8WBN9aLa3o47SoYctU6iLdj7wkGG5SV2hE5CgtD5,4,https://dogelonmars.com/img/logo.png,true
 Gecko,GECKO,6CNHDCzD5RkvBWxxyokQQNQPjFWgoHF94D7BmC73X6ZK,6,https://gateway.irys.xyz/2MOw1dmYBMp7XL-JylfJBKFsDZid_1xyK4cRjGYkzUY,true
 KNOB$,KNOB,5ritAPtFPqQtEFHcHVqNjR5oFNUJqcmgKtZyPd2AyLLy,9,https://i.postimg.cc/hPd4QCX6/knob-logo.jpg,true
-EURC,EURC,HzwqbKZw8HxMN6bF2yFZNrht3c2iXXzpKcFu7uBEDKtr,6,https://www.circle.com/hubfs/Brand/EURC/EURC-icon_128x128.png,true
\ No newline at end of file
+EURC,EURC,HzwqbKZw8HxMN6bF2yFZNrht3c2iXXzpKcFu7uBEDKtr,6,https://www.circle.com/hubfs/Brand/EURC/EURC-icon_128x128.png,true
+BOB,$BOB,HKUAxJzGTeeB6bJsgyiwPVncwoRyv8NBghUYEVzXEj1o,6,https://arweave.net/FGvSQj_eM1xgetBq3VkTH4dh-b8J4IaC6H1upUYy1V0,false
```
Comment (Timoon21) [inline validated-tokens.csv:1043]:
```suggestion
BOB,BOB,HKUAxJzGTeeB6bJsgyiwPVncwoRyv8NBghUYEVzXEj1o,6,https://arweave.net/FGvSQj_eM1xgetBq3VkTH4dh-b8J4IaC6H1upUYy1V0,true
```

**Review by Timoon21 — APPROVED** [review]

**Comment by Jymfletzfur** [comment]
insufficient volume

**Comment by demarco017** [comment]
It took forever but back when we requested we were putting volume its a
shame it took so long for a response, Thanks anyways

**Pull request closed without merging**